set.mm

```diff
@@ -647669,29 +647743,326 @@ standardize vectors to a length (norm) of one, but such definitions require
+    fltmul.c $e |- ( ph -> C e. CC ) $.
+    fltmul.n $e |- ( ph -> N e. NN0 ) $.
+    fltmul.1 $e |- ( ph -> ( ( A ^ N ) + ( B ^ N ) ) = ( C ^ N ) ) $.
+    $( A counterexample to FLT stays valid when scaled.  The hypotheses are
+       more general than they need to be for convenience.  (There does not seem
+       to be a standard term for Fermat or Pythagorean triples extended to any
+       ` N e. NN0 ` , hence the label is more about the context in which this
+       theorem is used).  (Contributed by SN, 20-Aug-2024.) $)
+    fltmul $p |- ( ph
+       -> ( ( ( S x. A ) ^ N ) + ( ( S x. B ) ^ N ) ) = ( ( S x. C ) ^ N ) ) $=
+      ( cexp co cmul caddc expcld adddid oveq2d mulexpd eqtr3d oveq12d 3eqtr4d
+      ) AEFMNZBFMNZONZUDCFMNZONZPNZUDDFMNZONZEBONFMNZECONFMNZPNEDONFMNAUDUEUGPN
+      ZONUIUKAUDUEUGAEFGKQABFHKQACFIKQRAUNUJUDOLSUAAULUFUMUHPAEBFGHKTAECFGIKTUB
+      AEDFGJKTUC $.
+  $}
+
+  ${
+    fltdiv.s $e |- ( ph -> S e. CC ) $.
+    fltdiv.0 $e |- ( ph -> S =/= 0 ) $.
+    fltdiv.a $e |- ( ph -> A e. CC ) $.
+    fltdiv.b $e |- ( ph -> B e. CC ) $.
+    fltdiv.c $e |- ( ph -> C e. CC ) $.
+    fltdiv.n $e |- ( ph -> N e. NN0 ) $.
+    fltdiv.1 $e |- ( ph -> ( ( A ^ N ) + ( B ^ N ) ) = ( C ^ N ) ) $.
+    $( A counterexample to FLT stays valid when scaled.  The hypotheses are
+       more general than they need to be for convenience.  (Contributed by SN,
+       20-Aug-2024.) $)
+    fltdiv $p |- ( ph
+       -> ( ( ( A / S ) ^ N ) + ( ( B / S ) ^ N ) ) = ( ( C / S ) ^ N ) ) $=
+      ( cexp co cdiv caddc expcld nn0zd expdivd expne0d divdird oveq12d 3eqtr4d
+      oveq1d eqtr3d ) ABFNOZEFNOZPOZCFNOZUHPOZQOZDFNOZUHPOZBEPOFNOZCEPOFNOZQODE
+      POFNOAUGUJQOZUHPOULUNAUGUJUHABFILRACFJLRAEFGLRAEFGHAFLSUAUBAUQUMUHPMUEUFA
+      UOUIUPUKQABEFIGHLTACEFJGHLTUCADEFKGHLTUD $.
+  $}
+
+  ${
+    flt0.a $e |- ( ph -> A e. CC ) $.
+    flt0.b $e |- ( ph -> B e. CC ) $.
+    flt0.c $e |- ( ph -> C e. CC ) $.
+    flt0.n $e |- ( ph -> N e. NN0 ) $.
+    flt0.1 $e |- ( ph -> ( ( A ^ N ) + ( B ^ N ) ) = ( C ^ N ) ) $.
+    $( A counterexample for FLT does not exist for ` N = 0 ` .  (Contributed by
+       SN, 20-Aug-2024.) $)
+    flt0 $p |- ( ph -> N e. NN ) $=
+      ( wcel cc0 wne cexp co caddc c1 exp0d wceq oveq2 cn0 cn c2 sn-1ne2 necomi
+      1p1e2 eqnetri a1i oveq12d 3netr4d eqeq12d syl5ibcom imp mteqand sylanbrc
+      elnnne0 ) AEUAKELMEUBKIAELBLNOZCLNOZPOZDLNOZAQQPOZQUSUTVAQMAVAUCQUFQUCUDU
+      EUGUHAUQQURQPABFRACGRUIADHRUJAELSZUSUTSZABENOZCENOZPOZDENOZSVBVCJVBVFUSVG
+      UTVBVDUQVEURPELBNTELCNTUIELDNTUKULUMUNEUPUO $.
+  $}
+
+  ${
+    fltabcoprm.s $e |- ( ph -> S e. NN ) $.
+    fltabcoprm.a $e |- ( ph -> A e. NN ) $.
+    fltabcoprm.b $e |- ( ph -> B e. NN ) $.
+    fltabcoprm.c $e |- ( ph -> C e. NN ) $.
+    fltabcoprm.n $e |- ( ph -> N e. NN0 ) $.
+    fltabcoprm.1 $e |- ( ph -> ( ( A ^ N ) + ( B ^ N ) ) = ( C ^ N ) ) $.
+    $( A counterexample to FLT implies a counterexample to FLT with ` A , B `
```

Comment:
At this point, you do not know that you obtain a Fermat or Pythagorean triple, since you do not know whether $A \wedge B | C$ (for Pythagorean triples, it turns out to be the case; I do not know in general, but in any case this theorem does not state it).

Reply:
I see you added the associated lemma `fltdvdsabdvdsc`: perfect. Resolving the conversation.